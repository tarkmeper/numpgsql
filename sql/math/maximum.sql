SELECT maximum('{1,4,5,6,7}'::integer[],5);
