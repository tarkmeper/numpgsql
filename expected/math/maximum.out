SELECT maximum('{1,4,5,6,7}'::integer[],5);
   maximum   
-------------
 {5,5,5,6,7}
(1 row)

